tests/build.gradle

```diff
@@ -16,6 +16,7 @@ tasks.withType(Test) {
 }
 
 task testLean(type: Test) {
+    exclude '**/*Java*'
```

Comment:
Do we need to exclude more now, because pushing the images takes some time? Would a different approach (e.g. Auto-building from Dockerhub) be a better solution then? (I know we'll have to rearrange things in our images for that, but maybe for the good?)